crypto/sessionkey.go

```diff
@@ -239,7 +239,15 @@ func encryptStreamWithSessionKey(
 	signEntity *openpgp.Entity,
 	config *packet.Config,
 ) (encryptWriter, signWriter io.WriteCloser, err error) {
-	encryptWriter, err = packet.SerializeSymmetricallyEncrypted(dataPacketWriter, config.Cipher(), sk.Key, config)
+	encryptWriter, err = packet.SerializeSymmetricallyEncrypted(
+		dataPacketWriter,
+		config.Cipher(),
```

Comment:
Why do we need the cipher both in here and in the ciphersuite ?

Reply:
In this case they're taken from the config, but in a key you could have different preferences for AEAD and for the normal cipher. So we need a generic interface that allows to pass different settings here

See [the SEIPD v1 preferences](https://www.ietf.org/archive/id/draft-ietf-openpgp-crypto-refresh-07.html#name-preferred-symmetric-ciphers) vs [the SEIPD v2 preferences](https://www.ietf.org/archive/id/draft-ietf-openpgp-crypto-refresh-07.html#name-preferred-aead-ciphersuites)

Reply:
But for a single call to SerializeSymetricallyEncrypted, you would use only one cipher right ?

Reply:
Yes, correct, the picked cipher depends whether AEAD is enabled or not